Google.Api.Generator.Tests/ProtoTests/ResourceNames/Testing.ResourceNames/ResourceNamesResourceNames.g.cs

```diff
@@ -70,6 +70,14 @@ public static bool TryParse(string singleResource, out SingleResourceName result
             }
         }
 
+        /// <summary>
+        /// Formats the IDs into the string representation of the <see cref="SingleResourceName"/> resource.
```

Comment:
The comment feels a little off due to the "resource" part at the end. Maybe just drop that? (Ditto in the returns.)

Reply:
Yes, I see what you mean. I'll do that.